catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -180,22 +183,10 @@ protected Object executeWithSubject() throws Exception {
             batchSize = 1;
         }
 
-        BundleContext bundleContext = getBundleContext();
-        if (!DEFAULT_TRANSFORMER_ID.equals(transformerId)) {
-            ServiceReference[] refs = null;
-
-            try {
-                refs = bundleContext.getServiceReferences(InputTransformer.class.getName(),
-                        "(|" + "(" + Constants.SERVICE_ID + "=" + transformerId + ")" + ")");
-            } catch (InvalidSyntaxException e) {
-                throw new IllegalArgumentException(
-                        "Invalid transformer transformerId: " + transformerId, e);
-            }
-
-            if (refs == null || refs.length == 0) {
-                throw new IllegalArgumentException("Transformer " + transformerId + " not found");
-            } else {
-                transformer = (InputTransformer) bundleContext.getService(refs[0]);
+        if (!FALLBACK_TRANSFORMER_ID.matches(transformerId)) {
+            transformer = getTransformer();
+            if (transformer == null) {
+                console.println(transformerId + " is an invalid input transformer.  Defaulting to Java Serialized Object.");
```

Comment:
Again I don't think we should default to any transformer if the transformer specified is invalid.